src/docs/components/docs-topbar/docs-topbar.tsx

```diff
@@ -8,12 +8,14 @@ import { Component, h, getAssetPath } from '@stencil/core'
   assetsDirs: ['assets'],
 })
 export class DocsTopbar {
+  private base = document.querySelector('base').href
+
   render() {
     return (
       <ld-header
         class="docs-topbar ld-theme-bubblegum"
         logo-title="Home"
-        logo-url="/"
+        logo-url={this.base}
```

Comment:
Shouldn't the absolute `logo-url` (actually a href attribute) depend on the base value in the head, anyway?

Reply:
It seems it doesn't. Currently setting an absolute URL with `/` results in a navigation to https://emdgroup-liquid.github.io/ which shows a 404.